AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +651,65 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    default_dash_layout = '''
+        {"nav" :
+              {"App Management":{"True":
```

Comment:
It's a good idea to indicate what each of the values mean in the data structure. There's nothing attached to "True" here to hint at what it's supposed to indicate.